dev-lang/ruby/ruby-2.3.1.recipe

```diff
@@ -0,0 +1,107 @@
+SUMMARY="A programming language focused on simplicity and productivity"
+DESCRIPTION="Ruby is a dynamic, reflective, object-oriented, general-purpose \
+programming language. It was designed and developed in the mid-1990s \
+by Yukihiro 'Matz' Matsumoto in Japan.
+
+Ruby embodies syntax inspired by Perl with Smalltalk-like features and \
+was also influenced by Eiffel and Lisp.  It supports multiple \
+programming paradigms, including functional, object oriented, and \
+imperative. It also has a dynamic type system and automatic memory \
+management. Therefore, it is similar in varying degrees to, Smalltalk, \
+Python, Perl, Lisp, Dylan, and CLU."
+HOMEPAGE="http://www.ruby-lang.org"
```

Comment:
You might wish to switch to https and add a trailing slash (to have a really valid URI):
```
HOMEPAGE="https://www.ruby-lang.org/"
```